staging/src/github.com/kubeedge/mapper-framework/pkg/grpcserver/device.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/avast/retry-go"
 	"k8s.io/klog/v2"
 	"reflect"
```

Comment:
please change the order of import to conform lint check.

Reply:
Yes, the order should be modified after the conflict is resolved @luomengY

Reply:
> please change the order of import to conform lint check.

ok

Reply:
> Yes, the order should be modified after the conflict is resolved @luomengY

I have made it.